README.md

```diff
@@ -64,13 +64,14 @@ When you run the above command, the following takes place:
 1. Registers and listens for events, executes the `Tasks` defined in the operation config. These tasks will be executed sequentially when the corresponding events occur.
 
 #### List of Environment Variables (You can use them directly in the startup command, export them, or add them to the `.env` file):
+
 | Name                        | Type    | Default   | Description                                                                                                                                                        | Example                                   |
-|-----------------------------|---------|-----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------|
+| --------------------------- | ------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------- | --- |
 | RESOURCE_CONFIG             | String  | ''        | Path to resource config yaml file.                                                                                                                                 | 'configs/resources/sample-resource.yml'   |
 | OPERATION_CONFIG            | String  | ''        | Path to operation config yaml file.                                                                                                                                | 'configs/operations/sample-operation.yml' |
 | INSTALLATION_ID             | String  | ''        | Installation Id of your GitHub App, must install the App to repositories before retrieving the id.                                                                 | '1234567890'                              |
 | ADDITIONAL_RESOURCE_CONTEXT | Boolean | false     | Setting true will let each resource defined in RESOURCE_CONFIG to call GitHub Rest API and GraphQL for more detailed context (ex: node_id). Increase startup time. | true / false                              |
-| SERVICE_NAME                | String  | 'default' | Set Service Name                                                                                                                                                   | 'My Service'                              |'
+| SERVICE_NAME                | String  | 'default' | Set Service Name                                                                                                                                                   | 'My Service'                              | '   |
```

Comment:
I think you need to remove the extra `'` and format again.

Reply:
These changes are the output of `npx prettier --write .`.

Reply:
I mean to say the original readme file has an extra `'` that should be removed which resulted in a wrong format.

Reply:
Updated can you check now?